### PR TITLE
Improved media handling

### PR DESCRIPTION
**Pull request by annando**
Depends on https://github.com/friendica/friendica/pull/10170

### CODE REVIEW

File: twitter/twitter.php [hunk 1]
```diff
@@ -1466,23 +1452,85 @@ function twitter_expand_entities($body, stdClass $status, $picture)
...
+ */
+function twitter_store_attachments(int $uriid, $post)
+{
+	if (!empty($post->extended_entities->media)) {
+		foreach ($post->extended_entities->media AS $medium) {
+			switch ($medium->type) {
+				case 'photo':
+					$attachment = ['uri-id' => $uriid, 'type' => Post\Media::IMAGE];
+
+					// Later store the large picture when we don't add them to the body anymore
```
Comment (MrPetovan) [inline twitter/twitter.php]: I don't understand this comment?

Comment (annando) [reply twitter/twitter.php]: Ah, now I know. Twitter does know several image sizes. We currently link the medium size, not the big one.

Comment (MrPetovan) [reply twitter/twitter.php]: What does "later" mean here? Later in the code or is it a `@TODO`?

Comment (annando) [reply twitter/twitter.php]: It's a `@TODO`

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**